tests/__init__.py

```diff
@@ -19,3 +18,81 @@ def setUpClass(cls):
     def skip_test_if_no_creds(self):
         if not self.has_access_key:
             self.skipTest("No boto credentials found")
+
+
+class TestConnection:
```

Comment:
Think this is fine but worth having a look at how we've mocked db outputs in the past for reference.

https://github.com/moj-analytical-services/dataengineeringutils3/blob/master/tests/mocks.py

Reply:
Ahh, this is much neater - thank you.